modules/serverless-build-func-kn.adoc

```diff
@@ -11,13 +11,13 @@ Before you can run a function, you must build the function project by using the
 .Example `func.yaml`
 [source,yaml]
 ----
-name: example-function
-namespace: default
-runtime: node
+name: <function_name>
+namespace: <function_namespace>
+runtime: <function_runtime>
 image: <image_from_registry>
 imageDigest: ""
 trigger: http
-builder: default
+builder: <builder_type>
 builderMap:
```

Comment:
Outside this PR but I think this should be `builders` not `builderMap`.

Reply:
Can you provide upstream docs for this please? It's not really within the scope of this Jira issue, but I think it might be better to expand the issue and improve the func.yaml docs, because this to me is confusing atm.

Reply:
https://github.com/knative-sandbox/kn-plugin-func/blob/main/docs/reference/func_yaml.md